common/types/message/message.go

```diff
@@ -99,22 +98,22 @@ func (e *Byte48) UnmarshalJSON(input []byte) error {
 // BatchTaskDetail is a type containing BatchTask detail.
 type BatchTaskDetail struct {
 	// use one of the string of EuclidFork / EuclidV2Fork
-	ForkName        string       `json:"fork_name"`
-	ChunkInfos      []*ChunkInfo `json:"chunk_infos"`
-	ChunkProofs     []ChunkProof `json:"chunk_proofs"`
```

Comment:
Would it still make sense to maintain an abstract proof type? Or do you think it's unnecessary overhead?